builtin/moon.pkg.json

```diff
@@ -18,6 +18,8 @@
     "int64_nonjs.mbt": ["not", "js"],
     "arraycore_js.mbt": ["js"],
     "arraycore_nonjs.mbt": ["not", "js"],
-    "array_nonjs_test.mbt": ["not", "js"]
+    "array_nonjs_test.mbt": ["not", "js"],
+    "stringbuilder_buffer.mbt": ["not", ["js", "wasm-gc"]],
+    "stringbuilder_concat.mbt": ["js", "wasm-gc"]
```

Comment:
I added `wasm-gc` to this list, since we are going to support `jsstring` in near future. 

But it still a question that can we assume every `wasm-gc` impies `jsstring` proposal?